src/cli/commands/start/start.ts

```diff
@@ -25,6 +28,7 @@ export async function start(options: SWACLIConfig) {
   // Code below doesn't have access to these environment variables which are defined later below.
   // Make sure this code (or code from utils) does't depend on environment variables!
 
+  const start = new Date().getTime();
```

Comment:
at all the places.. shall we name the variable start/end  as cmdStartTime/ cmdEndTime to be more clear on what these start/end refer to?